README.md

```diff
@@ -624,6 +675,91 @@ _You can enable the following settings in Xcode by running [this script](resourc
   ```
 
   </details>
+  
+* <a id='long-function-declaration'></a>(<a href='#long-function-declaration'>link</a>) **Separate [long](https://github.com/airbnb/swift#column-width) function declarations with line breaks before each argument label and before the return signature.** Put the open curly brace on the next line so the first executable line doesn't look like it's another parameter. [![SwiftFormat: wrapArguments](https://img.shields.io/badge/SwiftFormat-wrapArguments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapArguments) [![SwiftFormat: wrapMultilineStatementBraces](https://img.shields.io/badge/SwiftFormat-wrapMultilineStatementBraces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineStatementBraces)
```

Comment:
I think this and the following rule would be more at home in the `Functions` section than in the `Closures` section.

<img width="1066" alt="Screen Shot 2020-07-30 at 1 44 13 PM" src="https://user-images.githubusercontent.com/1791049/88972711-d16a6a80-d26a-11ea-9526-7a122452f4e0.png">

Reply:
Oops! Good catch